src/modules/leveling/awards/reactions.js

```diff
@@ -22,8 +23,13 @@ export default async function messageReactionAdd(client, messageReaction, user)
     const { message } = messageReaction;
     const userDto = userDb.get(user.id);
     const now = Date.now();
+    const donor = await getMember(user.id);
 
-    if (userDto.cooldown.get(reactionName) > now || message.author === user) {
+    if (
+        userDto.cooldown.get(reactionName) > now ||
+        message.author === user ||
```

Comment:
This should be donor
```suggestion
        message.member === donor ||
```

Reply:
That doesn't matter right? Could compare ids though...